packages/cli/src/commands/app/serve.ts

```diff
@@ -14,13 +14,19 @@
  * limitations under the License.
  */
 
-import { serveBundle } from '../../lib/bundler';
+import fs from 'fs-extra';
 import { Command } from 'commander';
+import { serveBundle } from '../../lib/bundler';
+import { paths } from '../../lib/paths';
 
 export default async (cmd: Command) => {
+  const pkgPath = paths.resolveTarget('package.json');
+  const pkg = await fs.readJson(pkgPath);
```

Comment:
would be better to do this as a part of serveBundle I think, that way it works for plugin:serve too

Reply:
done in #908

Reply:
👍 for doing these kind of changes in separate PRs